src/main/java/uk/gov/register/views/RecordView.java

```diff
@@ -35,27 +40,37 @@ public ObjectNode getRecordJson() {
         ObjectMapper objectMapper = Jackson.newObjectMapper();
         ObjectNode jsonNodes = objectMapper.convertValue(entry, ObjectNode.class);
         jsonNodes.remove("key");
-        jsonNodes.setAll(objectMapper.convertValue(itemView, ObjectNode.class));
+        ArrayNode items = jsonNodes.putArray("items");
+        itemViews.forEach( iv -> items.add(objectMapper.convertValue(iv, ObjectNode.class)));
         return jsonNodes;
     }
 
-    public Map<String, FieldValue> getContent() {
-        return itemView.getContent();
+    ArrayNode getFlatRecordJson() {
```

Comment:
In the entry view we just have the hashes which can be represented in CSV as a delimited string. But in the record view we want the combination of all the Entry and Item properties.